aten/src/ATen/native/mkldnn/Linear.cpp

```diff
@@ -72,7 +72,10 @@ Tensor mkldnn_linear(
       "mkldnn_linear: input needs to be mkldnn layout");
   if (self.scalar_type() == ScalarType::BFloat16) {
     TORCH_CHECK(mkldnn_bf16_device_check(),
-        "mkldnn_linear: bf16 path needs the cpu support avx512bw, avx512vl and avx512dq");
+        "mkldnn_linear: bf16 path needs the cpu support avx_ne_convert or avx512bw, avx512vl and avx512dq");
```

Comment:
Looks like mkldnn_matmul is supported on ARM devices. Are you sure about linter being the exception?

Reply:
I split bf16 check into bf16 check on x64 and on ARM.
I kept the previous message (adding a new isa `avx_ne_convert`), which also does not mention ARM. Do you think we should include ARM info. in all of such messages (include matmul, conv and deconv).
https://github.com/pytorch/pytorch/pull/99498/files#diff-dd15cec62ddb24d690b58e1902d8822347003437c9c4c4ae51f9c02e281fa33eR88